### ISSUE TITLE
Item - Black Chasm

### ISSUE THREAD

**Issue by Zedby2508** [WarEmu/WarBugs#8845]
Got this today, and I think it`s not proccing at all. I`ve been checking the combat log for a while, didn`t see the proc happening.

![image](https://cloud.githubusercontent.com/assets/25022364/21791874/4036e850-d6e6-11e6-8425-70dc6194ba39.png)

Thanks in advance

**Comment by Luth-RoR** [comment]
Do you have two epic weapons with damage procs equipped?

**Comment by Zedby2508** [comment]
Yeah I do, anything I should know about?

Dana 10. sij 2017. 14:24 osoba "Luth" <notifications@github.com> napisala
je:

Do you have two epic weapons with damage procs equipped?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/WarEmu/WarBugs/issues/8845#issuecomment-271574438>, or mute
the thread

**Comment by Luth-RoR** [comment]
Yes, probably because https://github.com/WarEmu/WarBugs/issues/6615#issuecomment-223384644

**Comment by Luth-RoR** [comment]
I just tested the spirit damage proc and it worked fine with the AP regen proc weapon in the other hand.

**Comment by Zedby2508** [comment]
Well I don't have the AP wep in the other hand unfortunately... Have one
with dot dmg over 9 seconds, can't remember the name. Do you know if this
will get fixed anytime soon?

Dana 10. sij 2017. 14:31 osoba "Luth" <notifications@github.com> napisala
je:
